src/benchcab/internal.py

```diff
@@ -13,7 +13,10 @@
 CONFIG_REQUIRED_KEYS = ["realisations", "modules"]
 
 # CMake module used for compilation:
-CMAKE_MODULE = "cmake/3.24.2"
+DEFAULT_MODULES = {
+    "cmake": "cmake/3.24.2",
+    "intel-compiler": "intel-compiler/2021.10.0",
```

Comment:
Yes, it is being used in `gen_codecov` stage, since `profmerge` and `codecov` have dependency

https://github.com/CABLE-LSM/benchcab/pull/298/files#diff-b8e7c82a7cc61f3622ab4ce5d454a3f8778ea2d558e997022d477be63665af52R55